### PR TITLE
Gradebook tweaks

### PR DESCRIPTION
**Pull request by mattwala**
Makes the following changes to the per-opportunity gradebook:
* Sorts by user id by default
* Shows all flow sessions
* Adds a "View page grades" button which allows all page grades to be shown in the gradebook

### CODE REVIEW

**Comment by mattwala** [comment]
* Also re-orders the columns so that the overall grade is the last column

**Comment by inducer** [comment]
Looks great! Thanks for the patch. This is now live on the UIUC instance.

One question: Why did you decide to show the page grade at the end of the page columns rather than at the beginning? Any particular reason or coincidence?

**Comment by mattwala** [comment]
I'm not sure what you mean. What page columns are you referring to?

**Comment by inducer** [comment]
The columns with the per-page grades precede the overall grade. Instinctively, I would have ordered them the other way around.

**Comment by mattwala** [comment]
Ah, I suppose if there is any reason why I chose it that way, it is because the "Overall grade" column is not that useful from the grader's perspective. As a grader I would want to have the page grades in the center and the "Overall grade" just uses up real estate and can be confused for a page grade.

**Comment by mattwala** [comment]
I suppose putting it as the first column would also be fine.

**Comment by inducer** [comment]
OIC. Fine as it is in this case. Thanks again!

File: course/templates/course/gradebook-by-opp.html [hunk 1]
```diff
@@ -109,12 +111,17 @@ <h3 class="start-well-title">{% trans "Modify many sessions at once" %}</h3>
           <a href="{% url "relate-view_single_grade" course.identifier participation.id opportunity.id %}"><span class="sensitive">{{ participation.user.username }}</span></a>
         </td>
         <td class="datacol">
-            <span class="sensitive">
-                {{ participation.user.get_full_name }}
```
Comment (dzhuang) [inline course/templates/course/gradebook-by-opp.html]: I am wondering why the user.get_full_name here is reverted back to 2 columns ;-(

Comment (inducer) [inline course/templates/course/gradebook-by-opp.html]: @dzhuang Sorry, missed that. @mattwala Could you send a PR reverting that?

Comment (mattwala) [inline course/templates/course/gradebook-by-opp.html]: Done. #198

Comment (inducer) [inline course/templates/course/gradebook-by-opp.html]: Merged.

**Pull request merged**